Megha/Milestone-21(Linked List)/Algorithms/doubly linked list/Implementing Doubly linked list.cp

```diff
@@ -0,0 +1,87 @@
+/* Doubly Linked List implementation */
```

Comment:
Well commented Code @Meghag5

Reply:
> Some of .DS_Store files are still their in your Pr. Delete those files, you can use git command for deleting such files using single command, Rather then going to each folder and delete .DS_Store files use git command.
> 
> ![1](https://user-images.githubusercontent.com/59874304/97774594-2b870e80-1b7f-11eb-8ae8-46453173a0e5.JPG)
> ![2](https://user-images.githubusercontent.com/59874304/97774595-2d50d200-1b7f-11eb-81b9-da3ab473801c.JPG)
I am not able to delete by using git command